client/packages/config/src/config.ts

```diff
@@ -1,26 +1,31 @@
-import { Capacitor } from '@capacitor/core';
-
 declare const API_HOST: string;
 
-// For production we get URL relative to web app
-const isProductionBuild = process.env['NODE_ENV'] === 'production';
-// http://localhost:8000 is default for remote server
-// API_HOST is available through webpack.DefinePlugin (i.e. webpack server --env API_HOST='http://localhost:8001')
+// For production, API is on the same domain/ip and port as web app, available through sub-route
```

Comment:
Comment should describe it pretty well, also merged config from index.ts here since it's very closely coupled